integrations/RollBar/browser.js

```diff
@@ -0,0 +1,496 @@
+            for (t = 0; t < l.length; ++t)
+              r[(a = l[t])] && r[a].prototype && n(e, r[a].prototype, o);
+          }
+        };
+      },
+      function (r, e, o) {
+        "use strict";
+        function n(r, e) {
+          (this.impl = r(e, this)),
+            (this.options = e),
+            (function (r) {
+              for (
+                var e = function (r) {
+                    return function () {
+                      var e = Array.prototype.slice.call(arguments, 0);
+                      if (this.impl[r]) return this.impl[r].apply(this.impl, e);
+                    };
+                  },
+                  o =
+                    "log,debug,info,warn,warning,error,critical,global,configure,handleUncaughtException,handleAnonymousErrors,handleUnhandledRejection,_createItem,wrap,loadFull,shimId,captureEvent,captureDomContentLoaded,captureLoad".split(
+                      ","
+                    ),
+                  n = 0;
+                n < o.length;
+                n++
+              )
+                r[o[n]] = e(o[n]);
+            })(n.prototype);
+        }
+        (n.prototype._swapAndProcessMessages = function (r, e) {
+          var o, n, t;
+          for (this.impl = r(this.options); (o = e.shift()); )
+            (n = o.method),
+              (t = o.args),
+              this[n] &&
+                "function" == typeof this[n] &&
+                ("captureDomContentLoaded" === n || "captureLoad" === n
+                  ? this[n].apply(this, [t[0], o.ts])
+                  : this[n].apply(this, t));
+          return this;
+        }),
+          (r.exports = n);
+      },
+      function (r, e, o) {
+        "use strict";
+        r.exports = function (r) {
+          return function (e) {
+            if (!e && !window._rollbarInitialized) {
+              for (
+                var o,
+                  n,
+                  t = (r = r || {}).globalAlias || "Rollbar",
+                  a = window.rollbar,
+                  l = function (r) {
+                    return new a(r);
+                  },
+                  i = 0;
+                (o = window._rollbarShims[i++]);
+
+              )
+                n || (n = o.handler),
+                  o.handler._swapAndProcessMessages(l, o.messages);
+              (window[t] = n), (window._rollbarInitialized = !0);
+            }
+          };
+        };
+      },
+    ]);
+    // End Rollbar Snippet
+    console.log(window.Rollbar);
+  }
+
+  isLoaded() {
+    logger.debug("===In isLoaded RollBar===");
+    return !!(window.Rollbar && isObject(window.Rollbar));
+  }
+
+  isReady() {
+    logger.debug("===In isReady RollBar===");
+
+    return !!window.Rollbar;
+  }
+
+  identify(rudderElement) {
+    logger.debug("===In RollBar Identify===");
+    const { message } = rudderElement;
+    const { userId } = message;
+    const { traits } = rudderElement.message.context;
+    if (userId === null || userId === undefined) return;
```

Comment:
why we are dumping anonymous users? Can't we merge anonymous users with identified ones?